src/test/ui/suggestions/missing-lifetime-specifier.stderr

```diff
@@ -171,8 +171,8 @@ LL | pub union Qux<'t, 'k, I> {
    |           ^^^ --  --
 help: add missing lifetime argument
    |
-LL |     static e: RefCell<HashMap<i32, Vec<Vec<Qux<'static, '_, i32>>>>> = RefCell::new(HashMap::new());
-   |                                                       ++++
+LL |     static e: RefCell<HashMap<i32, Vec<Vec<Qux<'static, 'static, i32>>>>> = RefCell::new(HashMap::new());
+   |                                                       +++++++++
```

Comment:
Spans looks slightly bigger than needed, but that was before that PR.

Reply:
What do you mean spans look larger? This is a suggestion, the fixup is to add `, 'static`.

Reply:
Ah, that is span to added suggestion, not old span, you right.